backend/typescript/rest/volunteerRoutes.ts

```diff
@@ -72,6 +91,25 @@ volunteerRouter.get("/:volunteerId?", async (req, res) => {
   }
 });
 
+volunteerRouter.get("/shifts/:volunteerId?", async (req, res) => {
```

Comment:
we won't have any queries for this endpoint, right?
```suggestion
volunteerRouter.get("/shifts/:volunteerId", async (req, res) => {
```